infrastructure/main.tf

```diff
@@ -0,0 +1,23 @@
+
+module "send-letter-service-api" {
+  source   = "git@github.com/hmcts/terraform-module-webapp.git"
+  product  = "${var.product}"
+  location = "${var.location_api}"
+  env      = "${var.env}"
+  ilbIp    = "${var.ilbIp}"
+
+  app_settings = {
+    REDIS_HOST                   = "${module.redis-cache.host_name}"
+    REDIS_PORT                   = "${module.redis-cache.redis_port}"
+    REDIS_PASSWORD               = "${module.redis-cache.access_key}"
```

Comment:
it may be worth cleaning up this spacing here ;)

Reply:
ok will create a new PR as this one is already merged now :)